docs/btc-methods.md

```diff
@@ -0,0 +1,73 @@
+# BTC Methods
+
+Here we document the Bitcoin methods that an account Snap may implement to
+support requests originated from dapps.
+
+## btc_sendmany
+
+This method is similar to the `sendmany` RPC method from Bitcoin Core, but its
+parameters are passed in an object instead of an array, and are named in
+camelCase. Also, dummy values aren't allowed.
+
+### Parameters
+
+- **Transaction intent (required)**
+  - Type: `object`
+  - Properties:
+    - `amounts`
+      - Description: A JSON object with recipient addresses and amounts.
+      - Type: `object`
+      - Properties:
+        - `[key]: string`: Address of the recipient
+        - `[value]: string`: Amount to send to the recipient in BTC
+    - `comment` (optional)
+      - Description: A comment.
+      - Type: `string`
+    - `subtractFeeFrom` (optional)
+      - Description: The fee will be equally deducted from the amount of each
+        selected address. Those recipients will receive less bitcoins than you
+        enter in their corresponding amount field. If no addresses are specified
+        here, the sender pays the fee.
+      - Type: `array`
+      - Items:
+        - Type: `string`
+    - `replaceable` (optional)
+      - Description: Allow this transaction to be replaced by a transaction
+        with higher fees via BIP 125.
+      - Type: `boolean`
+
+### Returns
+
+- **Response object:**
```

Comment:
I don't remember why we did wrap the transaction ID inside an object? Looking at https://github.com/MetaMask/keyring-api/blob/main/docs/evm-methods.md#returns-2

We could maybe use another wording here too like "Transaction context" maybe? "Transaction details"? Up to you, I just wanted to point that out

Reply:
> I don't remember why we did wrap the transaction ID inside an object?

To be able to extend the response if needed in the future.

> We could maybe use another wording here too? Like "Transaction context" maybe? "Transaction details"?

I think both would not be accurate, maybe `Transaction ID`.